source/concepts/harvesting.rst

```diff
@@ -51,16 +51,20 @@ To harvest locally, the account should provide the next properties in :propertie
     enableAutoHarvesting; bool; Set to true if delegated harvesting is enabled.; false
     maxUnlockedAccounts; uint32_t; Maximum number of delegated harvesting accounts.; 5
     delegatePrioritizationPolicy; harvesting::DelegatePrioritizationPolicy; Delegate harvester prioritization policy used to keep accounts once the node stores ``maxUnlockedAccounts``. Possible values are "Importance" or "Age".; Importance
-    beneficiaryPublicKey; string; Public key of the account receiving part of the harvested fee.; 0000000000000000000000000000000000000000000000000000000000000000
+    beneficiaryPublicKey; string; The public key of the account that will receive a percentage of the block fees defined by the harvestBeneficiaryPercentage.; 0000000000000000000000000000000000000000000000000000000000000000
+    harvestBeneficiaryPercentage; Percentage; Percentage of the block fees that will be given to the beneficiaryPublicKey account.
```

Comment:
harvestBeneficiaryPercentage, harvestNetworkFeeSinkPublicKey, and harvestNetworkPercentage are not under `config-harvesting.properties`, these are configurable per  network.